packages/moonstone/styles/colors-light.less

```diff
@@ -143,18 +143,19 @@
 
 // Input
 // ---------------------------------------
-@moon-input-text-color:               @moon-white;
```

Comment:
Wanted to verify that the specs use black text for the input text color in light skin. I didn't see the specs in the ticket but assume that's what was showing in zeppelin.